nilearn/surface/tests/test_surface.py

```diff
@@ -94,45 +92,34 @@ def test_load_surf_data_array():
     assert_array_equal(load_surf_data(data_squeeze), np.zeros((20, 3)))
 
 
-def test_load_surf_data_file_nii_gii(tmp_path):
-    # test loading of fake data from gifti file
-    fd_gii, filename_gii = tempfile.mkstemp(suffix=".gii", dir=str(tmp_path))
-    os.close(fd_gii)
+def test_load_surf_data_from_gifti_file(tmp_path):
```

Comment:
I personally like to have short docstrings to quickly assess what the test is testing.

Reply:
@bthirion how did this comment get here?

this file was not changed in this PR...

Reply:
line 95 is marked as changed in my Github interface...

Reply:
huh? this is strange: this file does not even show on my side

https://github.com/nilearn/nilearn/pull/4547/files

![Screenshot from 2024-10-15 12-12-13](https://github.com/user-attachments/assets/038b8fe0-318c-4b39-a4a3-99fa0d57b094)

Reply:
> huh? this is strange: this file does not even show on my side
> 
> https://github.com/nilearn/nilearn/pull/4547/files
> 
> ![Screenshot from 2024-10-15 12-12-13](https://private-user-images.githubusercontent.com/6961185/376564729-038b8fe0-318c-4b39-a4a3-99fa0d57b094.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg5OTc1NzgsIm5iZiI6MTcyODk5NzI3OCwicGF0aCI6Ii82OTYxMTg1LzM3NjU2NDcyOS0wMzhiOGZlMC0zMThjLTRiMzktYTRhMy05OWZhMGQ1N2IwOTQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTVUMTMwMTE4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OWQ2OGU1YzRhZmIxZmRkYThiMTNiZTAwMmE2YjU5ZmIyZDllYjIzZTYxNGMyZDU3ZjhiM2EzNDY5ZmEwYzg4OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.EwpCgSFPBDp86LFh5saTMlf--o_sp3ptjZV4RlMAtlw)

OK. It no long shows up in my interface either. NVM